lib/adal.js

```diff
@@ -438,9 +476,9 @@ var AuthenticationContext = (function () {
     };
 
     /**
-     * Acquire token from cache if not expired and available. Acquires token from iframe if expired.
+     * Acquires token from cache if not expired and available. Acquires token from iframe if expired.
```

Comment:
Acquires token from cache if there is a valid token present in the cache. Otherwise sends token request to AAD for a new token.